src/EditorFeatures/VisualBasicTest/Recommendations/PreprocessorDirectives/LoadDirectiveKeywordRecommenderTests.vb

```diff
@@ -0,0 +1,33 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Recommendations.PreprocessorDirectives
```

Comment:
Actually, it looks like this PR adds it, so I'm wrong. :smile:

IMO, this isn't worth recommending (I think it probably clutters completion more than it helps given how infrequently this will by typed).  We should at least make sure it isn't the first thing that shows up in completion.